modules/installation-guide/examples/system-variables.adoc

```diff
@@ -95,57 +95,55 @@ pass:[<!-- vale off -->]
+ `+CHE_INFRA_KUBERNETES_PVC_STRATEGY+`,"`+common+`","Defined which strategy will be used while choosing PVC for workspaces. Supported strategies: `common`::        All workspaces in the same {orch-namespace} will reuse the same PVC.        Name of PVC may be configured with `che.infra.kubernetes.pvc.name`.        Existing PVC will be used or a new one will be created if it does not exist. `unique`::        Separate PVC for each workspace's volume will be used.        Name of PVC is evaluated as `'{che.infra.kubernetes.pvc.name} + '-' + {generated_8_chars}'`.        Existing PVC will be used or a new one will be created if it does not exist. `per-workspace`::        Separate PVC for each workspace will be used.        Name of PVC is evaluated as `'{che.infra.kubernetes.pvc.name} + '-' + {WORKSPACE_ID}'`.        Existing PVC will be used or a new one will be created if it doesn't exist." 
+ `+CHE_INFRA_KUBERNETES_PVC_PRECREATE__SUBPATHS+`,"`+true+`","Defines whether to run a job that creates workspace's subpath directories in persistent volume for the `common` strategy before launching a workspace. Necessary in some versions of {orch-name} as workspace subpath volume mounts are created with root permissions, and therefore cannot be modified by workspaces running as a user (presents an error importing projects into a workspace in {prod-short}). The default is `true`, but should be set to `false` if the version of {orch-name} creates subdirectories with user permissions. Relevant issue: \https://github.com/kubernetes/kubernetes/issues/41638 the {prod-short} Operator that this property has effect only if the `common` PVC strategy used." 
+ `+CHE_INFRA_KUBERNETES_PVC_NAME+`,"`+claim-che-workspace+`","Defines the settings of PVC name for {prod-short} workspaces. Each PVC strategy supplies this value differently. See documentation for `che.infra.kubernetes.pvc.strategy` property" 
  `+CHE_INFRA_KUBERNETES_PVC_STORAGE__CLASS__NAME+`,"","Defines the storage class of Persistent Volume Claim for the workspaces. Empty strings means 'use default'." 
- `+CHE_INFRA_KUBERNETES_PVC_QUANTITY+`,"`+10Gi+`","Defines the size of Persistent Volume Claim of che workspace. Format described here: \https://docs.openshift.com/container-platform/4.4/storage/understanding-persistent-storage.html" 
- `+CHE_INFRA_KUBERNETES_PVC_JOBS_IMAGE+`,"`+centos:centos7+`","Pod that is launched when performing persistent volume claim maintenance jobs on OpenShift" 
+ `+CHE_INFRA_KUBERNETES_PVC_QUANTITY+`,"`+10Gi+`","Defines the size of Persistent Volume Claim of {prod-short} workspace. Format described here: \https://docs.openshift.com/container-platform/4.4/storage/understanding-persistent-storage.html" 
+ `+CHE_INFRA_KUBERNETES_PVC_JOBS_IMAGE+`,"`+registry.access.redhat.com/ubi8-minimal:8.3-230+`","Pod that is launched when performing persistent volume claim maintenance jobs on OpenShift" 
```

Comment:
note that this value will change periodically... perhaps just using `registry.access.redhat.com/ubi8-minimal` as a pointer to the latest one is better?